gamemodes/horde/gamemode/sv_economy.lua

```diff
@@ -228,6 +228,28 @@ function plymeta:Horde_DropMoney(amount)
     end
 end
 
+function plymeta:Horde_PayPlayer(plyToPay, amount)
+    if not plyToPay or not amount then return end
+    local plyToPay = tostring(plyToPay)
+    local amount = tonumber(amount)
+    local plyForMoney
+    local allPlys = player.GetAll()
+    for i=1, #allPlys do
+        local iterPly = allPlys[i]
+        local findString = string.find(string.lower(iterPly:GetName()), string.lower(plyToPay))
+        if findString then
+            plyForMoney = iterPly
+        end
+    end
+    if plyForMoney == nil then return end
```

Comment:
Maybe do a PrintMessage back to the caller to let them know that nobody was found